src/components/List/List.js

```diff
@@ -48,6 +56,11 @@ const List = React.forwardRef((props, ref) => {
                 e.preventDefault();
                 setSelectedIndex(nextSelectedIndex);
 
+                // Scroll view with a delay so this function would refer to the next highlighted item
+                setTimeout(() => {
+                    highlightedListItem.current.scrollIntoView(SCROLL_INTO_VIEW_SETTINGS);
```

Comment:
1. IE11 doesn't support scrollIntoView :(

2. What if component was unmounted before this timeout is executed?

Reply:
@antipin
1. Yeah, that's a pity, it doesn't get `SCROLL_INTO_VIEW_SETTINGS ` param but the scroll by itself is still working so we decided to stick with that.
2. I'll add `clearTimeout` function call before unmounting to be sure it will never happen